pfio/v2/local.py

```diff
@@ -4,9 +4,54 @@
 import shutil
 from typing import Optional
 
+try:
```

Comment:
Refer to: https://github.com/pfnet/pfio/pull/326#discussion_r1329411782

Reply:
Split into `pfio/v2/_profiler.py` and changed to import from `_profiler.py`.